### PR TITLE
Resolves #88 - histograms use wrong units

### LINKED ISSUE #88: New histogram feature with conversion error
Either my day is far too long today or PR #66 has a little mathematic error which I simply have not seen in my test environment because my setup was too little and fast. 

If I push some statsd timers to statsd and let them export with statsd_exporter all metrics are send to the +Inf bucket. statsd uses metrics in milliseconds per default but the histogram buckets are set to seconds. 

This came to my attention when I deployed the current master branch of statsd_exporter to one of our production host. To reproduce it, simply send some "realistic" metrics to statsd, like:

```
echo "test.timing:100|ms|@0.1" | nc -u -w0 127.0.0.1 8125
echo "test.timing:320|ms|@0.1" | nc -u -w0 127.0.0.1 8125
echo "test.timing:500|ms|@0.1" | nc -u -w0 127.0.0.1 8125
echo "test.timing:900|ms|@0.1" | nc -u -w0 127.0.0.1 8125
```

It will export to:

```
test_timing_bucket{job="debug.goethe.local",le="0.005"} 0
test_timing_bucket{job="debug.goethe.local",le="0.01"} 0
test_timing_bucket{job="debug.goethe.local",le="0.025"} 0
test_timing_bucket{job="debug.goethe.local",le="0.05"} 0
test_timing_bucket{job="debug.goethe.local",le="0.1"} 0
test_timing_bucket{job="debug.goethe.local",le="0.25"} 0
test_timing_bucket{job="debug.goethe.local",le="0.5"} 0
test_timing_bucket{job="debug.goethe.local",le="1"} 0
test_timing_bucket{job="debug.goethe.local",le="2.5"} 0
test_timing_bucket{job="debug.goethe.local",le="5"} 0
test_timing_bucket{job="debug.goethe.local",le="10"} 0
test_timing_bucket{job="debug.goethe.local",le="+Inf"} 4
```

Can someone confirm this or am I doing something wrong?

### PR DESCRIPTION
**Pull request by drawks**
Convert milliseconds to seconds before passing into prometheus_client histogram

### CODE REVIEW

**Comment by grobie** [comment]
Thanks! Could you try and add a test for this?

**Comment by drawks** [comment]
@grobie Added test as requested.

File: exporter_test.go [hunk 1]
```diff
@@ -74,6 +75,44 @@ func TestInvalidUtf8InDatadogTagValue(t *testing.T) {
...
 
+type MockHistogram struct {
+	prometheus.Metric
+	prometheus.Collector
+	value float64
+}
+
+func (h *MockHistogram) Observe(n float64) {
+	h.value = n
+
```
Comment (SuperQ) [inline exporter_test.go]: Extra newline.

File: exporter_test.go [hunk 2]
```diff
@@ -15,6 +15,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
```
Comment (grobie) [inline exporter_test.go]: Please keep the usual order of imports in go: first packages from stdlib, newline, then external packages (both lists must be sorted).

File: exporter_test.go [hunk 3]
```diff
@@ -74,6 +75,44 @@ func TestInvalidUtf8InDatadogTagValue(t *testing.T) {
...
+	ex := NewExporter(&metricMapper{}, true)
+	ex.mapper.Defaults.TimerType = timerTypeHistogram
+
+	// Close channel to signify we are done with the listener after a short period.
+	go func() {
+		time.Sleep(time.Millisecond * 100)
+		close(events)
+	}()
+	mock := &MockHistogram{}
+	ex.Histograms.Elements[7787632782521330630] = mock
```
Comment (grobie) [inline exporter_test.go]: Where does this magic number come from?

Comment (drawks) [inline exporter_test.go]: That is the hash of the metric, it is how all the metrics are stored internally. I suppose I could call out the hash function to generate it.

Comment (SuperQ) [inline exporter_test.go]: That would make it more clear.

**Comment by drawks** [comment]
@grobie @SuperQ I've made changes to address your line markups

**Comment by grobie** [comment]
Thanks!

**Pull request merged**